superpixel_classification/SuperpixelClassification/SuperpixelClassificationBase.py

```diff
@@ -426,22 +426,27 @@ def createFeatures(self, gc, folderId, annotationName, featureFolderId, patchSiz
         prog.items([item for item, _, _ in itemsAndAnnot])
         results = {}
         futures = []
+        featureFiles = list(gc.listFile(featureFolderId))
```

Comment:
`gc.listFiles` takes an _item_ id (not a folder id).  I think you want to use `gc.listItem`.